Unreal/DazStudioPlugin/DzUnrealAction.cpp

```diff
@@ -56,51 +56,53 @@ void DzUnrealAction::executeAction()
 	 // Create and show the dialog. If the user cancels, exit early,
 	 // otherwise continue on and do the thing that required modal
 	 // input from the user.
-
-	 if (dzScene->getNumSelectedNodes() != 1)
-	 {
-		  QMessageBox::warning(0, tr("Error"),
-				tr("Please select one Character or Prop to send."), QMessageBox::Ok);
-		  return;
-	 }
-
-	 // Create the dialog
-	 DzUnrealDialog* dlg = new DzUnrealDialog(mw);
-
-	 // If the Accept button was pressed, start the export
-	 if (dlg->exec() == QDialog::Accepted)
-	 {
-		 // Collect the values from the dialog fields
-		 CharacterName = dlg->assetNameEdit->text();
-		 ImportFolder = dlg->intermediateFolderEdit->text();
-		 CharacterFolder = ImportFolder + "\\" + CharacterName + "\\";
-		 CharacterFBX = CharacterFolder + CharacterName + ".fbx";
-		 CharacterBaseFBX = CharacterFolder + CharacterName + "_base.fbx";
-		 CharacterHDFBX = CharacterFolder + CharacterName + "_HD.fbx";
-		 AssetType = dlg->assetTypeCombo->currentText().replace(" ", "");
-		 MorphString = dlg->GetMorphString();
-		 Port = dlg->portEdit->text().toInt();
-		 ExportMorphs = dlg->morphsEnabledCheckBox->isChecked();
-		 ExportSubdivisions = dlg->subdivisionEnabledCheckBox->isChecked();
-		 MorphMapping = dlg->GetMorphMapping();
-		 ShowFbxDialog = dlg->showFbxDialogCheckBox->isChecked();
-		 ExportMaterialPropertiesCSV = dlg->exportMaterialPropertyCSVCheckBox->isChecked();
-		 SubdivisionDialog = DzUnrealSubdivisionDialog::Get(dlg);
-		 FBXVersion = dlg->fbxVersionCombo->currentText();
-
-		 if (AssetType == "SkeletalMesh" && ExportSubdivisions)
-		 {
-			 // export base mesh
-			 ExportBaseMesh = true;
-			 SubdivisionDialog->LockSubdivisionProperties(false);
-			 Export();
-		 }
-
-		 ExportBaseMesh = false;
-		 SubdivisionDialog->LockSubdivisionProperties(ExportSubdivisions);
-		 Export();
-
-	 }
+     if (getNonInteractiveMode() > 0) {
+
+         if (dzScene->getNumSelectedNodes() != 1)
+         {
+             QMessageBox::warning(0, tr("Error"),
```

Comment:
Line 63: The two calls to QMessageBox::warning() in the executeAction() method should only be called if NonInteractiveMode == 0.

Please see implementation of this logic here:
https://github.com/danielbui78/UDTU/blob/5e4e6e398261b7bc5c6b7fd9310d5df6623a05d2/Unity/DazStudioPlugin/DzUnityAction.cpp#L100-L101